cluster/src/main/scala/org/apache/spark/memory/SnappyUnifiedMemoryManager.scala

```diff
@@ -580,6 +619,8 @@ object SnappyUnifiedMemoryManager extends Logging {
       math.max(getMaxHeapMemory / 20, 500L * 1024L * 1024L))
   }
 
+  private val DEFAULT_EVICTION_PERCENT = 80f
```

Comment:
My experience has been that GC always cleans up stuff before reaching 95%. So we can at least increase the critical to that especially if heap size is greater than say 4g (on Gem side too). Lets test more for eviction.